src/coreclr/jit/compiler.cpp

```diff
@@ -2612,7 +2612,9 @@ void Compiler::compInitOptions(JitFlags* jitFlags)
     opts.compJitAlignLoopAdaptive       = true;
     opts.compJitAlignLoopBoundary       = DEFAULT_ALIGN_LOOP_BOUNDARY;
     opts.compJitAlignLoopMinBlockWeight = DEFAULT_ALIGN_LOOP_MIN_BLOCK_WEIGHT;
+    opts.compJitAlignLoopMaxCodeSize    = DEFAULT_MAX_LOOPSIZE_FOR_ALIGN;
```

Comment:
This is a potential uninitialized variable in Release without this. Turns out in release adaptive is always true so we don't use it, but IMO it's better to set it in case that changes.